### DISCUSSION TITLE
Proxy for GPRadio (location spoofing)

### DISCUSSION THREAD

**Discussion by R0GGER** [f1multiviewer/issue-tracker#416] [Feature requests] [answered]
It would be great to have an option to connect to a socks5 proxy for GPRadio.
With a socks5 you can connect to GPRadio without VPN, it allows users to change their virtual location (also known as location spoofing). 

More info: https://nordvpn.com/nl/blog/socks5-proxy/

**Comment by f1multiviewer** [comment, marked as answer]
Sorry, but VPNs are against the rules (in our Discord, and on our subreddit) for a reason. If we'd introduce this, it'd be a sure way to get us DMCA'd. Sorry, but we won't be adding this.
